post_processing_stages/CMakeLists.txt

```diff
@@ -25,6 +25,12 @@ else()
     message(WARNING "OpenCV not found, some stages will not be built!")
 endif()
 
+set(ENABLE_TFLITE)
+if (ENABLE_TFLITE)
+    set(SRC ${SRC} object_classify_tf_stage.cpp)
+    set(TARGET_LIBS ${TARGET_LIBS} tensorflow-lite atomic)
+endif()
+
```

Comment:
Do we want to have a mechanism where we check if the tensorflow-lite library is found, similar to what we do for OpenCV? 

Is atomic needed?

Reply:
I agree that we should check if it's there, though the default is not to include TF stages so it's more of a nice-to-have.

libatomic did seem to be necessary, but it's worth checking again seeing as various bits of code have been chucked out!

Reply:
Atomic actually not needed anymore!

Reply:
We still seem to have `atomic` on line 31... was it supposed to be removed?